structures/multistep.js

```diff
@@ -0,0 +1,13 @@
+const s = require('superstruct')
+
+const MAX_PKG = 50
```

Comment:
@lvenier Agreed with this constant?

Reply:
I think that should not be a contact but a default value.
pls check https://github.com/yoctu/rtech-struct/blob/main/structures/auction.js#L13 and we can use a config value "config.shaq.pkgmax"

Reply:
You're right.